spec/ship_spec.rb

```diff
@@ -0,0 +1,30 @@
+require './lib/ship'
+
+Rspec.describe Ship do
+  it 'exists' do
+    cruiser = Ship.new("Cruiser", 3)
+    expect(cruiser).to be_an_instance_of(Ship)
+  end
+
+  it 'has attributes' do
+    cruiser = Ship.new("Cruiser", 3)
+    expect(cruiser.name).to eq("Cruiser")
+    expect(cruiser.length).to eq(3)
+    expect(cruiser.health).to eq(3)
+    expect(cruiser.sunk?).to eq(false)
```

Comment:
I think that sunk? needs its own test since it's testing a method(behavior) rather than something innate defined in the initialize method(attribute). Will you please separate these and push that change to your branch?

Reply:
How's this? 

  it 'can sink' do
    cruiser = Ship.new("Cruiser", 3)
    cruiser.hit
    cruiser.hit
    cruiser.hit
    expect(sunk?).to eq(true)
  end